erpnext/stock/doctype/material_request_item/material_request_item.json

```diff
@@ -177,6 +177,7 @@
    "fieldtype": "Column Break"
   },
   {
+   "allow_on_submit": 1,
```

Comment:
If we want to allow this then we also need to validate by adding same validation in `before_update_after_submit` methods on material request.


ref:
https://github.com/frappe/erpnext/blob/79a51b24812725a541c3accf68fdde004582da5d/erpnext/stock/doctype/material_request/material_request.py#L59-L62

https://github.com/frappe/frappe/blob/a886d793f29fb3c291620958286b52b3665c1ebc/frappe/model/document.py#L955-L961

Reply:
I think Validate does not run on Update, can you please explain the reason of updating Validate code? @ankush

Reply:
It **should** run on update, right now it doesn't :) that's what I am asking for adding.